dcp_test.go

```diff
@@ -21,8 +24,7 @@ import (
 	"github.com/Trendyol/go-dcp/config"
 
 	"github.com/Trendyol/go-dcp/couchbase"
-	"github.com/couchbase/gocbcore/v10"
-	"github.com/testcontainers/testcontainers-go"
+	// _ "github.com/emrygun/go-dcp-tracing-otel"
```

Comment:
can we remove this comment and all comments from functions headers without description? after thats LGTM 🚀